<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>بگرد و بیاب</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
    background-color:rgb(15, 15, 15) ;
}
QFrame#frame_main{
    background-color: rgb(34, 34, 34);
}
QPushButton{
    color: rgb(255, 255, 255);
    background-color: rgb(19, 63, 59);
    border-radius: 10px;
}
QPushButton:hover{
    background-color: rgba(39, 92, 86, 0.938);
}
QLabel{
    background-color: rgb(34, 34, 34);
}
QComboBox{
    color: rgb(255, 255, 255);
    background-color: rgba(26, 80, 75, 0.288);
}
QDateEdit{
    background-color: rgba(26, 80, 75, 0.288);
    color: white;
}
QSpinBox{
    color: white;
    background-color: rgba(26, 80, 75, 0.288);;
}
QLineEdit{
    color: rgb(255, 255, 255);
    background-color: rgb(19, 63, 59);
    border-radius: 10px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>50</number>
      </property>
      <property name="topMargin">
       <number>50</number>
      </property>
      <property name="rightMargin">
       <number>50</number>
      </property>
      <property name="bottomMargin">
       <number>50</number>
      </property>
      <item row="0" column="0">
       <widget class="QFrame" name="frame_main">
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_14" stretch="0,5">
           <item>
            <spacer name="verticalSpacer_9">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_13">
             <item>
              <layout class="QGridLayout" name="gridLayout_6" columnminimumwidth="0">
               <property name="sizeConstraint">
                <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
               </property>
               <property name="verticalSpacing">
                <number>20</number>
               </property>
               <item row="1" column="0">
                <layout class="QGridLayout" name="gridLayout_7" rowstretch="1,1,1">
                 <property name="leftMargin">
                  <number>20</number>
                 </property>
                 <property name="topMargin">
                  <number>20</number>
                 </property>
                 <property name="rightMargin">
                  <number>20</number>
                 </property>
                 <property name="bottomMargin">
                  <number>20</number>
                 </property>
                 <item row="0" column="0">
                  <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,2">
                   <property name="spacing">
                    <number>20</number>
                   </property>
                   <property name="leftMargin">
                    <number>50</number>
                   </property>
                   <property name="rightMargin">
                    <number>50</number>
                   </property>
                   <item>
                    <spacer name="horizontalSpacer_9">
                     <property name="orientation">
                      <enum>Qt::Orientation::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QSplitter" name="splitter_3">
                     <property name="orientation">
                      <enum>Qt::Orientation::Horizontal</enum>
                     </property>
                     <widget class="QPushButton" name="myrooms">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>200</width>
                        <height>35</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>B Nazanin</family>
                        <pointsize>15</pointsize>
                       </font>
                      </property>
                      <property name="cursor">
                       <cursorShape>PointingHandCursor</cursorShape>
                      </property>
                      <property name="text">
                       <string>اتاق های من</string>
                      </property>
                     </widget>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item row="1" column="0">
                  <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,2,2,2,2">
                   <property name="spacing">
                    <number>25</number>
                   </property>
                   <property name="sizeConstraint">
                    <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
                   </property>
                   <property name="leftMargin">
                    <number>50</number>
                   </property>
                   <property name="rightMargin">
                    <number>50</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout">
                     <property name="spacing">
                      <number>5</number>
                     </property>
                     <item>
                      <spacer name="verticalSpacer">
                       <property name="orientation">
                        <enum>Qt::Orientation::Vertical</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>20</width>
                         <height>40</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                     <item>
                      <widget class="QLabel" name="label">
                       <property name="font">
                        <font>
                         <family>B Nazanin</family>
                         <pointsize>13</pointsize>
                        </font>
                       </property>
                       <property name="text">
                        <string>حداکثر قیمت</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout">
                       <item>
                        <spacer name="horizontalSpacer">
                         <property name="orientation">
                          <enum>Qt::Orientation::Horizontal</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                        </spacer>
                       </item>
                       <item>
                        <widget class="QLineEdit" name="max_price">
                         <property name="minimumSize">
                          <size>
                           <width>250</width>
                           <height>35</height>
                          </size>
                         </property>
                         <property name="font">
                          <font>
                           <family>B Nazanin</family>
                           <pointsize>12</pointsize>
                          </font>
                         </property>
                         <property name="layoutDirection">
                          <enum>Qt::LayoutDirection::RightToLeft</enum>
                         </property>
                         <property name="placeholderText">
                          <string>  اگر خالی باشد محدودیت قیمتی وجود ندارد</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <spacer name="verticalSpacer_2">
                       <property name="orientation">
                        <enum>Qt::Orientation::Vertical</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>20</width>
                         <height>40</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_9">
                     <property name="spacing">
                      <number>2</number>
                     </property>
                     <item>
                      <spacer name="verticalSpacer_19">
                       <property name="orientation">
                        <enum>Qt::Orientation::Vertical</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>20</width>
                         <height>40</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_9">
                       <property name="font">
                        <font>
                         <family>B Nazanin</family>
                         <pointsize>13</pointsize>
                        </font>
                       </property>
                       <property name="text">
                        <string>تعداد افراد</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QSpinBox" name="passengers">
                       <property name="minimumSize">
                        <size>
                         <width>0</width>
                         <height>40</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <spacer name="verticalSpacer_20">
                       <property name="orientation">
                        <enum>Qt::Orientation::Vertical</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>20</width>
                         <height>40</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_10">
                     <property name="spacing">
                      <number>2</number>
                     </property>
                     <item>
                      <spacer name="verticalSpacer_21">
                       <property name="orientation">
                        <enum>Qt::Orientation::Vertical</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>20</width>
                         <height>40</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_10">
                       <property name="font">
                        <font>
                         <family>B Nazanin</family>
                         <pointsize>13</pointsize>
                        </font>
                       </property>
                       <property name="text">
                        <string>تاریخ خروج</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QDateEdit" name="date_exit">
                       <property name="minimumSize">
                        <size>
                         <width>0</width>
                         <height>40</height>
                        </size>
                       </property>
                       <property name="maximumDateTime">
                        <datetime>
                         <hour>12</hour>
                         <minute>59</minute>
                         <second>59</second>
                         <year>2027</year>
                         <month>12</month>
                         <day>30</day>
                        </datetime>
                       </property>
                       <property name="minimumDateTime">
                        <datetime>
                         <hour>16</hour>
                         <minute>30</minute>
                         <second>0</second>
                         <year>2025</year>
                         <month>12</month>
                         <day>30</day>
                        </datetime>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <spacer name="verticalSpacer_22">
                       <property name="orientation">
                        <enum>Qt::Orientation::Vertical</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>20</width>
                         <height>40</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_11">
                     <property name="spacing">
                      <number>2</number>
                     </property>
                     <item>
                      <spacer name="verticalSpacer_23">
                       <property name="orientation">
                        <enum>Qt::Orientation::Vertical</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>20</width>
                         <height>40</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_11">
                       <property name="font">
                        <font>
                         <family>B Nazanin</family>
                         <pointsize>13</pointsize>
                        </font>
                       </property>
                       <property name="text">
                        <string>تاریخ ورود</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QDateEdit" name="date_enter">
                       <property name="minimumSize">
                        <size>
                         <width>0</width>
                         <height>40</height>
                        </size>
                       </property>
                       <property name="maximumDateTime">
                        <datetime>
                         <hour>12</hour>
                         <minute>59</minute>
                         <second>59</second>
                         <year>2027</year>
                         <month>12</month>
                         <day>30</day>
                        </datetime>
                       </property>
                       <property name="minimumDateTime">
                        <datetime>
                         <hour>16</hour>
                         <minute>30</minute>
                         <second>0</second>
                         <year>2025</year>
                         <month>12</month>
                         <day>30</day>
                        </datetime>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <spacer name="verticalSpacer_24">
                       <property name="orientation">
                        <enum>Qt::Orientation::Vertical</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>20</width>
                         <height>40</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_12">
                     <property name="spacing">
                      <number>2</number>
                     </property>
                     <item>
                      <spacer name="verticalSpacer_25">
                       <property name="orientation">
                        <enum>Qt::Orientation::Vertical</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>20</width>
                         <height>40</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_12">
                       <property name="font">
                        <font>
                         <family>B Nazanin</family>
                         <pointsize>13</pointsize>
                        </font>
                       </property>
                       <property name="text">
                        <string>مقصد هتل(شهر)</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QComboBox" name="des">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="minimumSize">
                        <size>
                         <width>0</width>
                         <height>40</height>
                        </size>
                       </property>
                       <property name="cursor">
                        <cursorShape>PointingHandCursor</cursorShape>
                       </property>
                       <item>
                        <property name="text">
                         <string>اصفهان</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>تبریز</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>چابهار</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>رشت</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>کیش</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>مشهد</string>
                        </property>
                       </item>
                      </widget>
                     </item>
                     <item>
                      <spacer name="verticalSpacer_26">
                       <property name="orientation">
                        <enum>Qt::Orientation::Vertical</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>20</width>
                         <height>40</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                 <item row="2" column="0">
                  <layout class="QHBoxLayout" name="horizontalLayout_7">
                   <property name="topMargin">
                    <number>12</number>
                   </property>
                   <item>
                    <spacer name="horizontalSpacer_7">
                     <property name="orientation">
                      <enum>Qt::Orientation::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QPushButton" name="main_find">
                     <property name="minimumSize">
                      <size>
                       <width>200</width>
                       <height>50</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <family>B Nazanin</family>
                       <pointsize>16</pointsize>
                      </font>
                     </property>
                     <property name="cursor">
                      <cursorShape>PointingHandCursor</cursorShape>
                     </property>
                     <property name="text">
                      <string>جست و جو</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_8">
                     <property name="orientation">
                      <enum>Qt::Orientation::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item row="2" column="0">
                <widget class="QFrame" name="frame">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>150</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::Shape::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Shadow::Raised</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1075</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
